src/content/docs/style-guide/writing-guidelines/user-related-language-guidelines.mdx

```diff
@@ -89,7 +105,7 @@ We recommend linking to the [explanation of default groups](/docs/accounts/accou
 
 ## Philosophy for user permissions/roles language [#user-language-philosophy]
 
-Before writing about permission-related requirements, it may help if you learn about [the relationships between groups, roles, and capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#type-role-group). 
+Before learning how to write about permission-related requirements, it may help to learn about [the relationships between groups, roles, and capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#type-role-group). 
 
```

Comment:
I wonder if strength of claim should be stronger here? I would honestly recommend that folks do not write about permissions related requirements without at least a solid basic understanding of groups, roles, and capabilities. How about: `Before learning about permission-related requirements, it's a good idea to make sure you understand the basics of the relationships between groups, roles, and capabilities`.